crates/amaru/tests/conformance/stake-distributions/Makefile

```
@@ -93,3 +97,13 @@ upload: check-network-dir check-aws-env ## Upload missing local 'epoch_*.json.zs
 		uploaded=$$((uploaded + 1)); \
 	done; \
 	echo "Uploaded $$uploaded file(s); skipped $$skipped already present in the bucket."
+	@$(MAKE) --no-print-directory -f "$(THIS_MAKEFILE)" manifest
+
+manifest: check-network-dir ## Add the local 'epoch_*.json' and 'epoch_*.json.zst' fixtures to MANIFEST
```

checkmake (0.3.2)

[warning] Target body for "manifest" exceeds allowed length of 5 lines (7).

(maxbodylength)